test/uk/gov/hmrc/traderservices/journey/CreateCaseJourneyStateFormatsSpec.scala

```diff
@@ -185,7 +185,7 @@ class CreateCaseJourneyStateFormatsSpec extends UnitSpec {
               freightType = Some(ExportFreightType.RORO),
               contactInfo = Some(
                 ExportContactInfo(
-                  contactName = "Full Name",
+                  contactName = Some("Full Name"),
```

Comment:
There's another one on line 68 (stops the tests from compiling) but just needs the same change